swap-covenant/justfile

```diff
@@ -0,0 +1,45 @@
+build:
+    cargo build
+
+gen: build gen-schema
+
+gen-schema:
+	START_DIR=$(pwd); \
+	for f in ./packages/*; do \
+		echo "generating schema"; \
+		cd "$f"; \
+		CMD="cargo run --example schema"; \
+		eval ${CMD} > /dev/null; \
+		rm -rf ./schema/raw; \
+		cd "$START_DIR"; \
+	done
+
+test:
+    cargo test
+
+lint:
+	cargo +nightly clippy --all-targets -- -D warnings && cargo +nightly fmt --all --check
```

Comment:
You can remove `+nightly` here because we are not using it, and shouldn't use it actually (I think)